package.json

```diff
@@ -64,7 +64,11 @@
     "terser-webpack-plugin": "^1.1.0",
     "thread-loader": "^2.1.3",
     "webpack": "^4.43.0",
-    "webpack-bundle-analyzer": "^3.3.2"
+    "webpack-bundle-analyzer": "^3.3.2",
+    "swr": "^0.3.2",
+    "prop-types": "^15.7.2",
+    "object-hash": "^2.0.3",
+    "localforage": "^1.7.4"
```

Comment:
why were these moved?

Reply:
Because `_app.jsx` needs to have all its dependencies inside `dependencies` and not `devDependencies`.

`swr`, `prop-types` are used now by `_app.jsx`. `object-hash` and `localforage` are now not in the list because I refactored the one piece of code where they were used and it's not really necessary anymore.

Reply:
makes sense, thanks for explaining!